autoload/jedi.vim

```diff
@@ -403,7 +403,7 @@ function! jedi#configure_call_signatures()
         autocmd InsertEnter <buffer> let g:jedi#first_col = s:save_first_col()
     endif
     autocmd InsertEnter <buffer> let s:show_call_signatures_last = [0, 0, '']
-    autocmd InsertLeave <buffer> PythonJedi jedi_vim.clear_call_signatures()
```

Comment:
F***. I actually thought this might be an issue, but at the same time I thought you might have changed it already.